src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -39,6 +39,9 @@ private void ageCheck(Integer age) {
     }
 
     private void passwordLengthCheck(Integer passwordLength) {
+        if (passwordLength == null) {
+            throw new RuntimeException("Age can't be null!");
```

Comment:
You forgot to change the message. Also, you can’t use ```passwordLength``` variable here. What happens if you call method  ```.length()```  for a null object?